### ISSUE TITLE
No GPU utilization with QBatchNorm2d

### ISSUE THREAD

**Issue by Sahel13** [ispamm/hTorch#10]
Hi,

When using the QBatchNorm2d layer, there is no GPU utilization, and consequently, model training is taking absurdly long times. Without this layer, everything works as expected. Kindly check this and let me know how this can be fixed. You can recreate this issue by testing any convolutional model that uses QBatchNorm2d.

**Comment by giorgiozannini** [comment]
Hi, thanks for spotting the bug. I'll check it as soon as possible!

**Comment by giorgiozannini** [comment]
I'm sorry for the late reply I was very busy. I cannot seem to reproduce the missing cuda computation on `QBatchNorm2d`. It is however somewhat slow to the CPU version when the tensor in input is small. This is the code I used for testing:
```
bn = QBatchNorm2d(1)
bn_cuda = QBatchNorm2d(1).cuda()
input = torch.randn((2, 4, 100, 100))
```
```
%timeit bn(input)
```
```
%timeit bn_cuda(input.cuda())
```

Being a manual implementation of the batch normalization it is, unfortunately, much slower than a `nn.BatchNorm2d` since the operations are much more computationally expensive (because of the covariance matrix, the inverse etc...).

**Comment by Sahel13** [comment]
Yes, GPU utilization is happening now. Seems to have been an issue at my end.
